### DISCUSSION TITLE
How to make a fork work?

### DISCUSSION THREAD

**Discussion by amongURedImposter** [SHSGames/shsgames.github.io#341] [Q&A] [answered]
I put it on pages using a fork, but didn't make it my github homepage. 
Now the problem, the index.html links directly to my github homepage when it needs to link to the repository.
so if I go to the source code, and click on icon.png, it links to myname.github.io/icon.png (doesnt exists) instead of myname.github.io/repository/icon.png (where it should go). this is happening to all the files, so the website appears blank. I also can't have it be my homepage. 
In my other repositories, src="icon.png" leads to the repository's page instead of the homepage. 
Does anyone know how to fix this? 
Thanks

**Comment by JoshMerlino** [comment, marked as answer]
1. In the github pages section of the fork, configure it to use the `public_html` branch as the webroot.
![image](https://user-images.githubusercontent.com/31785395/164241501-7ad69eb8-e377-40b5-a209-fc8cf819b41e.png)
2. Modify the `base` in the manifest to be the pathname that the mirror lives on.
    * For example, mine is `https://joshmerlino.github.io/SHSGames/` so my base would be set to `/SHSGames/` as seen in https://github.com/JoshMerlino/SHSGames/commit/0527b30b3935ea493ad64715cd5e3bfc4e48a87d

Then after GitHub Actions rebuilds the site, it should work on your mirror. However there may be limited PWA functionality due to it not being on the root domain.

**Comment by maverickteckh** [comment]
Also if it isn't working for you then you need to delete your fork and fork it again and make sure "only copy master branch" is unchecked.
Good Luck! 
-Maxwick456
